spec/factories/users.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    name 'MyString'
+    date_of_birth '2018-02-06 16:21:16'
+    gender 'male'
+    email 'pinkyrout10@gmail.com'
```

Comment:
Each time we create a `User` using this factory it will have the same email Id `pinkyrout10@gmail.com` which is wrong.
The factory is hard coded. Please fix this.

Reply:
should we use Faker or logical increment in email ids{pinky1@gmail.com, pinky2@gmail.com}?

Reply:
Use `Faker`